packages/web/src/components/gcds-icon/gcds-icon.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://unpkg.com/font-awesome/css/font-awesome.min.css');
+@import url('font-awesome.min.css');
```

Comment:
The import was set up to always load the newest version of font awesome so that we don't have to manually update it ourselves. Do you have any concerns about doing it that way or why did you change it?

Reply:
Yes1 The latest unpkg font awesome version is not up to date anymore. Maybe we should consider the npm package instead.

Reply:
Oh weird, unpkg should always have the latest version.
Let's use the npm package instead then.